### PR TITLE
Feature/tools

### PR DESCRIPTION
**Pull request by pasyukevich**
What was done: 

-  Eslint with callstack config;
-  Eslint errors fixed
-  Added prettier with config

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "plugins": ["prettier"],
```
Comment (jaworek) [inline .eslintrc:2]: You may not need to setup prettier as it is already done in the callstack eslint config.

File: src/SectionList.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          },
+        ]}
+      >
+        {children}
+      </Text>
+    </View>
+  );
+}
+
+export const SectionList = (): JSX.Element => {
```
Comment (jaworek) [inline src/SectionList.tsx:44]: [nit] I would omit `JSX.Element` return type as this is inferred by TS automatically. Even in the case where something incorrect is returned, using it in other component will result in an error.

**Review by jaworek — APPROVED** [review]

File: src/SectionList.tsx (hunk 2 above)
Comment (pasyukevich) [reply src/SectionList.tsx:44]: Sure, this part will be removed with the components update

File: .eslintrc (hunk 1 above)
Comment (pasyukevich) [reply .eslintrc]: Updated

**Pull request merged**